src/components/grid-aware/Footer/InfoBlock/InfoBlock.module.css

```diff
@@ -0,0 +1,44 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.infoBlock {
+  background-color: var(--color-midnight-blue);
+  color: var(--color-gray-300);
+  display: grid;
+  font-family: var(--font-family-lato);
+  font-size: 14px;
+  font-weight: 300;
+  grid-column: 1 / span 12;
```

Comment:
There's only 3 columns in the parent grid (which is the bleed), so it doesn't make sense to span 12 columns here. That said, maybe we don't even want to use grid here, since this element is not actually grid aligned. Could you just get rid of the `display: grid` on the parent element? In fact, you could probably collapse the `bleedWrapper` and `infoBlock` into one element.

Reply:
ok